contracts/ITitleEscrow.sol

```diff
@@ -56,6 +59,8 @@ interface ITitleEscrow {
   /// @param newBeneficiary The address of the new beneficiary
   /// @param newHolder The address of the new holder
   function transferToNewEscrow(address newBeneficiary, address newHolder) external;
+
+  function approveNewTransferTargets(address newBeneficiary, address newHolder) external;
```

Comment:
I notice that all functions above uses @param and not this one, should we add some doc ?

Reply:
fixed lol